.readthedocs.yaml

```diff
@@ -0,0 +1,20 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+   configuration: docs/conf.py
```

Comment:
Um, this PR deletes that file. Is it needed?

Reply:
`conf.py` is not deleted?

Reply:
You're quite right. Sorry for getting this wrong :/